platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -846,6 +853,7 @@ public void snapshot(final Path snapshotDirectory) throws IOException, IllegalSt
         }
     }
 
+    /*
```

Comment:
This method is commented out. Do you still need it?

Reply:
Nice catch! This functionality is moved to VirtualMap level, no longer needed in MerkleDb. I'll remove the comment.